### PR TITLE
LPS-53734 SF

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by rotty3000** [comment]
ci:retest

**Comment by brianchandotcom** [comment]
ci:retest

**Comment by brianchandotcom** [comment]
@peterfellwock can you resend? 2 test failures.  Thx.

File: portal-impl/test/integration/com/liferay/portal/kernel/parsers/bbcode/bundle/bbcodetranslatorutil/TestBBCodeTranslator.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	}
+
+	@Override
+	public String[][] getEmoticons() {
+		return null;
+	}
+
+	@Override
+	public String[] getEmoticonSymbols() {
+		return new String [] {"1", "2", "3", "4"};
```
Comment (rotty3000) [inline portal-impl/test/integration/com/liferay/portal/kernel/parsers/bbcode/bundle/bbcodetranslatorutil/TestBBCodeTranslator.java]: @peterfellwock the issue is with these array brackets :(

Remove the space and you'll be good!

**Pull request closed without merging**